src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -58,12 +58,16 @@ public with sharing class STG_PanelRD_CTRL extends STG_Panel {
         Savepoint sp = Database.setSavepoint();
         try {
         	List<SObject> toUpdate = new List<SObject>();
-        	if(STG_Panel.stgService.stgRD != null)
+        	
+        	if(STG_Panel.stgService.stgRD != null && !Test.isRunningTest()) {
         		toUpdate.add(STG_Panel.stgService.stgRD);
+        	}
+        	
         	if(rdSched != null) {
                 rdSched.Active__c = !isRDScheduleDisabled;
                 toUpdate.add(rdSched);
```

Comment:
Well, this is interesting... If we were following the pattern we are following everywhere else, we would have needed to wrap the " update toUpdate" (just a little lower in the code) statement in !isRunningTest. We would also have had to wrap the saveAll method in saveAll with that condition.

However, these were not giving row lock errors, so I didn't add it. It seems like the test runner is treating controllers different from other tests, as saving settings from them don't produce these row lock errors.

We are calling saveSettings from a controller in quite a few places without having to do anything about it:

![screen shot 2015-01-29 at 9 50 06 am](https://cloud.githubusercontent.com/assets/1894238/5960588/3a2d4a38-a79c-11e4-850b-6cb414c899bf.png)

I would leave it as is (why put the condition if we don't need it), but we can talk about it when we meet.